auth/src/main/java/com/firebase/ui/auth/util/ui/FlowUtils.java

```diff
@@ -16,17 +17,20 @@ private FlowUtils() {
     }
 
     public static boolean handleError(@NonNull HelperActivityBase activity, @NonNull Exception e) {
+        if (!(e instanceof StatefulException) || ((StatefulException) e).isUsed()) {
+            return false;
+        }
+        ((StatefulException) e).setUsed(true);
```

Comment:
Maybe the `setUsed` could be automatically called inside `getPendingIntent()` or `getIntent()`  so that way you can really only get them once?

Reply:
Yeah, I actually did that to start but it felt kinda weird. If you had that idea too, then it can't be _that_ weird. 😆 SGTM!